src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -73,18 +74,22 @@ const privateRecordReleasesSchema = Object.assign({}, treatments.items.propertie
 
 const formConfig = {
   urlPrefix: '/',
+  intentToFileUrl: '/21-526EZ', // TODO: update with ITF endpoint
```

Comment:
We should write a ticket to capture this `TODO` if there isn't one written already.

Reply:
Sounds good, @SMLuthi, do we know what this will be, or should we create a ticket to set this up once we do?